### DISCUSSION TITLE
Nested Upsert With Json Array

### DISCUSSION THREAD

**Discussion by BishoyNarouz** [prisma/prisma#6179] [Q&A]
how can i use nested upsert with json array 
for example : 
```ts
 prisma.user.upsert({
where: {id: userId},
update: { 
username: username, 
email: email ,
profiles : [ {where: {id: profileId} , update: {}, create: {}  },  {where: {id: profileId} , update: {}, create: {}  } ]
},
create:  {
username: username, 
email: email ,
profiles : [ {where: {id: profileId} , update: {}, create: {}  },  {where: {id: profileId} , update: {}, create: {}  } ]
}
})
```
how to upsert nested json array (profiles)?
Thanks

**Comment by ryands17** [comment]
@BishoyNarouz Could you share your schema?

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
